aries-site/src/layouts/main/Layout.js

```diff
@@ -52,16 +60,41 @@ export const Layout = ({
   } = getPageDetails(titleProp);
   const layout = isLanding ? 'plain' : pageLayout;
 
+  const defaultFeedback = {
+    'like-rating': '',
+    'star-rating': 0,
+    'text-area': '',
+  };
+
   const MainContentWrapper = isLanding ? Fragment : PageContent;
-  const size = useContext(ResponsiveContext);
+  const breakpoint = useContext(ResponsiveContext);
+  const [open, setOpen] = useState(false);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+  const [value, setValue] = useState(defaultFeedback);
+  const [isSucessful, setIsSuccess] = useState(false);
+
+  const close = () => {
+    setTimeout(() => {
+      setOpen(false);
+      setIsSuccess(false);
+    }, 2000);
+  };
+
+  const onSubmit = event => {
+    event.preventDefault();
+    console.log('response from user', event.value);
```

Comment:
I assume this `console.log` will be removed once we connect with qualtrics API?

Reply:
you are correct